requirements.txt

```diff
@@ -7,6 +7,6 @@ sphinx-issues
 sphinx_rtd_theme
 rst2pdf
 flake8
-pyvsc
+pyvsc==0.2.3.20201006.1
```

Comment:
Is there any issue with the latest version (0.2.5)?

Reply:
As of now, I have not encountered any issues with the new release.
Reverted this back to the original state.